src/crypto/PersistentStorageOperationalKeystore.cpp

```diff
@@ -310,4 +325,38 @@ void PersistentStorageOperationalKeystore::ReleaseEphemeralKeypair(Crypto::P256K
     Platform::Delete<Crypto::P256Keypair>(keypair);
 }
 
+CHIP_ERROR PersistentStorageOperationalKeystore::MigrateOpKeypairForFabric(FabricIndex fabricIndex,
+                                                                           OperationalKeystore & operationalKeystore) const
+{
+    VerifyOrReturnError(mStorage, CHIP_ERROR_INCORRECT_STATE);
```

Comment:
```suggestion
    VerifyOrReturnError(mStorage != nullptr, CHIP_ERROR_INCORRECT_STATE);
```
nit: but in the upstream devs often ask for full condition to have the code more self-explanatory.